supertokens_python/supertokens.py

```diff
@@ -388,6 +398,125 @@ async def get_users(  # pylint: disable=no-self-use
 
         return UsersResponse(users, next_pagination_token)
 
+    async def create_user_id_mapping(  # pylint: disable=no-self-use
+        self,
+        supertokens_user_id: str,
+        external_user_id: str,
+        external_user_id_info: Optional[str] = None,
```

Comment:
this is a mistake in the node SDK, but you need to add an extra argument to the function: `force: Optional[bool] = false` as well, and pass it in the API (see swagger spec: https://app.swaggerhub.com/apis/supertokens/CDI). So even the user exposed functions will change

Reply:
The use of `force` from a user's point of view is: 
- if they try and create a mapping for a user who already has a session, or already has some metadata / role, then the core will throw an error - not allowing you to create a mapping.
- but if you pass `force` as true in that case, it will allow you to make a mapping.

Reply:
Fixed.